### DISCUSSION TITLE
Blitz "lite"

### DISCUSSION THREAD

**Discussion by johtso** [blitz-js/blitz#2567] [Q&A]
I've only recently been checking out Blitz and I was wondering if there any way to set up a project that just has the core query/RPC functionality and nothing else?

I'm building a project that I intend to be fairly stateless (no server side sessions), uses a remote service for authentication, and only uses a Redis database as a job queue.. no other database or schema.

The RPC functionality would be fantastic as it would remove all the boilerplate of client/server communication for a setup that's really just a 1 client / 1 server situation.. but I'm worried that blitz is designed to do much more for you and would end up being a bad fit.

Would be interested in any thoughts!

**Comment by flybayer** [comment]
Yes, great idea! Can you open an issue for this (vs discussion)? That way it is tracked and not lost in discussions :)
